monte_carlo.py

```diff
@@ -70,18 +72,19 @@ def transport_neutron(mat, bounds, tallies, fission_banks, first_round, mesh):
     while neutron.alive:
         
         # sample neutron distance to collision
-        neutron_distance = sample_distance(mat)
+        neutron_distance = sample_distance( \
+                materials[mesh.get_material(neutron.get_cell())])
     
         # track neutron until collision or leakage
         while neutron_distance > 0:
 
-            # get cell boundaries
-            cell_boundaries = { \
-                    'min': mesh.get_cell_min(neutron.get_position_vector()),
-                    'max': mesh.get_cell_max(neutron.get_position_vector())}
-            
             # nudge neutron forward
-            neutron.move(TINY_MOVE)
+            #neutron.move(TINY_MOVE)
```

Comment:
The nudging isn't needed anymore because we're checking for the neutron's direction of travel when it's on the edge of a cell. I forgot to delete the comment.